src/materials.jl

```diff
@@ -82,6 +87,7 @@ function Material(::Type{Val{:flibe}})
     mat.type = [IMAS._blanket_]
     mat.density = (; temperature::Float64) -> (temperature - 273.15) * -0.425 + 2245.5 # fitted from Vidrio et al, J. Chem. Eng. Data 2022, 67, 12
     mat.cost_kg = 43.0 # source: https://fti.neep.wisc.edu/fti.neep.wisc.edu/presentations/mes_zpinch_tofe06.pdf, slide 20
+    # mat.cost_kg = 1000 # source: https://github.com/Woodruff-Scientific-Ltd/PyFECONS/blob/e8650deda30b1fb4171ca6d7777d240ed5a44a44/pyfecons/materials.py
```

Comment:
I think it's a good idea to leave this here commented as you did and continue running with our old value because the blanket is often the most expensive thing and this is a really significant cost difference...